docs/en/ingest-management/tab-widgets/install-layout.asciidoc

```diff
@@ -8,6 +8,8 @@ Main {agent} configuration
 Main {agent} {fleet} configuration
 `/Library/Elastic/Agent/elastic-agent.log`::
```

Comment:
Is this correct? Should I update this to show the path to the json log instead (it contains more info)? What is the difference between elastic-agent.log and elastic-agent-json.log? Should we describe the two logs here?

Reply:
I would document the .json log.  
/Library/Elastic/Agent/data/elastic-agent-*/logs/elastic-agent-json.log`::

You could also note that logs are numbered log.1 and log.2 and log.3 and so on as they are rotated off of the main file.

for Linux .tar.gz install of Agent, the main Agent log file is at:
/opt/Elastic/Agent/data/elastic-agent-e8aa6d/logs/elastic-agent-json.log

for deb and rpm install of Agent, the main Agent log file is at:
/var/lib/elastic-agent/data/elastic-agent-*/logs/elastic-agent-json.log

for Windows install of Agent, the main Agent log file is at:
'C:\Program/ Files\Elastic\Agent\data\elastic-agent-*\logs\elastic-agent-json.log

this should really help folks, nicely done.